### PR TITLE
[EN/ZH] MXB-3rd: Initial article

### PR DESCRIPTION
**Pull request by kj415j45**

### CODE REVIEW

File: wiki/Tournaments/MXB/3/en.md [hunk 1]
```diff
@@ -0,0 +1,159 @@
+---
+tags:
+- MXB
+---
+# The 3rd MXB osu!mania 4K Tournament
+
+![MXB 3rd Banner](img/banner.jpg)
```
Comment (fajar13k) [inline wiki/Tournaments/MXB/3/en.md]:
```suggestion
![MXB 3rd Logo](img/banner.jpg)
```
Change filename to `logo.jpg`

File: wiki/Tournaments/MXB/3/en.md [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+
+**Novice Mania Cup 3rd** (***MXB 3rd***) is a one-on-one osu!mania 4K specified Chinese regional tournament hosted by the MXB team. It is the 3rd installment of the MXB osu!mania 4K Tournament.
```
Comment (fajar13k) [inline wiki/Tournaments/MXB/3/en.md]:
```suggestion
The **Novice Mania Cup 3rd** (***MXB 3rd***) is a one-on-one osu!mania 4K specified Chinese regional tournament hosted by the MXB team. It is the 3rd installment of the MXB osu!mania 4K Tournament.
```

File: wiki/Tournaments/MXB/3/en.md [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+| Finals - Week 1 | 2019-06-01/2019-06-02 |
+| Finals - Week 2 | 2019-06-08/2019-06-09 |
+
+## Prizes
+
+Prizes for this tournament are sponsored by [IA daisuki](https://osu.ppy.sh/users/11238501)
+
+| Placing | Prize(s) |
+| :-: | :-- |
+| ![Gold Crown](/wiki/shared/GCrown.png "1st place") | 2 months of osu!supporter tag, unique userpage badge |
```
Comment (fajar13k) [inline wiki/Tournaments/MXB/3/en.md]:
```suggestion
| ![Gold Crown](/wiki/shared/GCrown.png "1st place") | 2 months of osu!supporter tag, unique profile badge |
```
perhaps

File: wiki/Tournaments/MXB/3/en.md [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+| Map Selectors | ![][flag_CN] [ExNeko](https://osu.ppy.sh/users/7590894), ![][flag_CN] [IA daisuki](https://osu.ppy.sh/users/11238501), ![][flag_US] [qqqant](https://osu.ppy.sh/users/8976576), ![][flag_CN] [Toki 0w0](https://osu.ppy.sh/users/10392201), ![][flag_CN] [-\[Ulazis\]-](https://osu.ppy.sh/users/11144363) |
+| Streaming Referees | ![][flag_CN] [Flow Light](https://osu.ppy.sh/users/6705256), ![][flag_CN] [IA daisuki](https://osu.ppy.sh/users/11238501), ![][flag_CN] [NimaPro](https://osu.ppy.sh/users/11524571), ![][flag_CN] [Rasis](https://osu.ppy.sh/users/1209363) |
+| Commentators | ![][flag_CN] [crazybrother](https://osu.ppy.sh/users/8045083), ![][flag_CN] [Flow Light](https://osu.ppy.sh/users/6705256), ![][flag_CN] [NimaPro](https://osu.ppy.sh/users/11524571), ![][flag_CN] [Rasis](https://osu.ppy.sh/users/1209363), ![][flag_CN] [-\[Ulazis\]-](https://osu.ppy.sh/users/11144363) |
+| Wiki | ![][flag_CN] [IA daisuki](https://osu.ppy.sh/users/11238501), ![][flag_CN] [kj415j45](https://osu.ppy.sh/users/9367540) |
+
+## Links
+
+- [Challonge](https://challonge.com/zh_CN/MXB3rd)
```
Comment (fajar13k) [inline wiki/Tournaments/MXB/3/en.md]:
```suggestion
- [Challonge brackets](https://challonge.com/zh_CN/MXB3rd)
```
No spreadsheet yet?

File: wiki/Tournaments/MXB/3/en.md [hunk 5]
```diff
@@ -0,0 +1,159 @@
...
+- Livestream
+  - [Flow Light](https://live.bilibili.com/73668)
+  - [IA daisuki](https://live.bilibili.com/4353181)
+  - [NimaPro](https://live.bilibili.com/12166146)
+  - [Rasis](https://live.bilibili.com/5333)
+
+## Participants
+
+525426806, H-ShJie, Star0v0, wuzhin, Anonie, shizehao, s_w_w, KirigayaSnow, Yutto007, RW233, solende, xiao ba, Chinese_Psy, 1ess__, qiuzhilin, Earnestlyqq, a794151336, amxmodx, bouquetww, KaffuCchino, yuki hirahira, HerrySun, Blue_Potion, magical Sao
```
Comment (fajar13k) [inline wiki/Tournaments/MXB/3/en.md]: Maybe do tables for participants?

File: wiki/Tournaments/MXB/3/en.md [hunk 6]
```diff
@@ -0,0 +1,159 @@
...
+
+## Mappools
+
+Mappools will be released 1 week before each stage.
```
Comment (fajar13k) [inline wiki/Tournaments/MXB/3/en.md]:
```suggestion
*Mappools will be released 1 week before each stage.*
```

File: wiki/Tournaments/MXB/3/en.md [hunk 7]
```diff
@@ -0,0 +1,159 @@
...
+
+## Rules
+
+### Tournament Organization
```
Comment (fajar13k) [inline wiki/Tournaments/MXB/3/en.md]:
```suggestion
### Tournament Organisation
```

File: wiki/Tournaments/MXB/3/en.md [hunk 8]
```diff
@@ -0,0 +1,159 @@
+---
+tags:
+- MXB
```
Comment (Joehuu) [inline wiki/Tournaments/MXB/3/en.md]: Keyword already in article.
```suggestion
```

File: wiki/Tournaments/MXB/3/en.md [hunk 9]
```diff
@@ -0,0 +1,159 @@
+---
```
Comment (Joehuu) [inline wiki/Tournaments/MXB/3/en.md]:
```suggestion
```

File: wiki/Tournaments/MXB/3/en.md [hunk 10]
```diff
@@ -0,0 +1,159 @@
...
+tags:
```
Comment (Joehuu) [inline wiki/Tournaments/MXB/3/en.md]:
```suggestion
```

File: wiki/Tournaments/MXB/3/en.md [hunk 11]
```diff
@@ -0,0 +1,159 @@
...
+- MXB
+---
```
Comment (Joehuu) [inline wiki/Tournaments/MXB/3/en.md]:
```suggestion
```

File: wiki/Tournaments/MXB/3/en.md [hunk 12]
```diff
@@ -0,0 +1,159 @@
...
+# The 3rd MXB osu!mania 4K Tournament
```
Comment (Joehuu) [inline wiki/Tournaments/MXB/3/en.md]: Use consistent tournament names. Assuming this is the right one from looking at the logo.
```suggestion
# Novice Mania Cup 3rd
```

File: wiki/Tournaments/MXB/3/en.md [hunk 13]
```diff
@@ -0,0 +1,159 @@
...
+
+![MXB 3rd Logo](img/logo.jpg)
+
+The **Novice Mania Cup 3rd** (***MXB 3rd***) is a one-on-one osu!mania 4K specified Chinese regional tournament hosted by the MXB team. It is the 3rd installment of the MXB osu!mania 4K Tournament.
+​​
+## Tournament Schedule
+
+| Event | Timestamp |
+| --: | :-- |
+| Registration | 2019-04-05/2019-04-17 |
```
Comment (Joehuu) [inline wiki/Tournaments/MXB/3/en.md]:
```suggestion
| Registration Phase | 2019-04-05/2019-04-17 |
```

File: wiki/Tournaments/MXB/3/en.md [hunk 14]
```diff
@@ -0,0 +1,159 @@
...
+| Live Drawings | 2019-05-05 19:00 (UTC+8) |
+| Group Stage | 2019-05-11/2019-05-12 |
+| Quarterfinals | 2019-05-18/2019-05-19 |
+| Semifinals | 2019-05-25/2019-05-26 |
+| Finals - Week 1 | 2019-06-01/2019-06-02 |
+| Finals - Week 2 | 2019-06-08/2019-06-09 |
+
+## Prizes
+
+Prizes for this tournament are sponsored by [IA daisuki](https://osu.ppy.sh/users/11238501)
```
Comment (Joehuu) [inline wiki/Tournaments/MXB/3/en.md]:
```suggestion
Prizes for this tournament are sponsored by [IA daisuki](https://osu.ppy.sh/users/11238501).
```

File: wiki/Tournaments/MXB/3/en.md [hunk 15]
```diff
@@ -0,0 +1,159 @@
...
+| Placing | Prize(s) |
+| :-: | :-- |
+| ![Gold Crown](/wiki/shared/GCrown.png "1st place") | 2 months of osu!supporter tag, unique profile badge |
+| ![Silver Crown](/wiki/shared/SCrown.png "2nd place") | 1 month of osu!supporter tag |
+| ![Bronze Crown](/wiki/shared/BCrown.png "3rd place") | 15 CNY |
+
+## Organisation
+
+| Position | Members |
+| :-: | :-- |
```
Comment (Joehuu) [inline wiki/Tournaments/MXB/3/en.md]:
```suggestion
| :-- | :-- |
```

File: wiki/Tournaments/MXB/3/en.md [hunk 16]
```diff
@@ -0,0 +1,155 @@
...
+
+| Placing | Prize(s) |
+| :-: | :-- |
+| ![Gold Crown](/wiki/shared/GCrown.png "1st place") | 2 months of osu!supporter tag, unique profile badge |
+| ![Silver Crown](/wiki/shared/SCrown.png "2nd place") | 1 month of osu!supporter tag |
+| ![Bronze Crown](/wiki/shared/BCrown.png "3rd place") | 15 CNY |
+
+## Organisation
+
+| Position | Members |
```
Comment (Joehuu) [inline wiki/Tournaments/MXB/3/en.md]:
```suggestion
| Position | Member(s) |
```

File: wiki/Tournaments/MXB/3/en.md [hunk 17]
```diff
@@ -0,0 +1,155 @@
+# Novice Mania Cup 3rd
+
+![MXB 3rd Logo](img/logo.jpg)
+
+The **Novice Mania Cup 3rd** (***MXB 3rd***) is a one-on-one osu!mania 4K specified Chinese regional tournament hosted by the MXB team. It is the 3rd installment of the MXB osu!mania 4K Tournament.
```
Comment (TPGPL) [inline wiki/Tournaments/MXB/3/en.md]:
```suggestion
The **Novice Mania Cup 3rd** (***MXB 3rd***) is a one-on-one osu!mania 4K specified Chinese regional tournament hosted by the MXB team. It is the third installment of the MXB osu!mania 4K Tournament.
```

File: wiki/Tournaments/MXB/3/en.md [hunk 18]
```diff
@@ -0,0 +1,155 @@
+# Novice Mania Cup 3rd
```
Comment (Joehuu) [inline wiki/Tournaments/MXB/3/en.md]: Search actually prioritizes tags, so add it back.
```suggestion
---
tags:
- MXB
---
# Novice Mania Cup 3rd
```

File: wiki/Tournaments/MXB/3/zh.md [hunk 19]
```diff
@@ -0,0 +1,144 @@
+# 第三届萌新杯
```
Comment (Joehuu) [inline wiki/Tournaments/MXB/3/zh.md]:
```suggestion
---
tags:
- MXB
---
# 第三届萌新杯
```

File: wiki/Tournaments/MXB/3/en.md [hunk 20]
```diff
@@ -0,0 +1,166 @@
...
+- [Challonge brackets](https://challonge.com/zh_CN/MXB3rd)
+- Livestream
+  - [Flow Light](https://live.bilibili.com/73668)
+  - [IA daisuki](https://live.bilibili.com/4353181)
+  - [NimaPro](https://live.bilibili.com/12166146)
+  - [Rasis](https://live.bilibili.com/5333)
+
+## Participants
+
+|  |  |  |  |
```
[the remark this answers is not in the document]
Comment (kj415j45) [reply wiki/Tournaments/MXB/3/en.md]: Markdown table must have a header

**Pull request merged**